ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java

```diff
@@ -33,7 +32,7 @@
  * A class consisting of common util methods useful for tests.
  */
 public class TestUtils {
-  public static final int TTL_SECS = 7200;
+  public static final long TTL_SECS = TimeUnit.DAYS.toSeconds(7);
```

Comment:
Why increase this to 7 days?

Reply:
TTL updates won't happen if the blob's expiry is less than a day way (`StoreConfig`) - so `7200` won't cut it. 7 days was just a random selection